fiona/model.py

```diff
@@ -197,6 +201,10 @@ def __init__(self, coordinates=None, type=None, geometries=None, **data):
         )
         super().__init__(**data)
 
+    def __repr__(self):
+        kvs = [f"{k}={_coord_repr.repr(v)}" for k, v in self.items() if v is not None]
+        return "fiona.Geometry({})".format(", ".join(kvs))
```

Comment:
Is `fiona.` necessary? `` here is just `Geometry`, so perhaps not?
```suggestion
        return "{}({})".format(__class__.__name__, ", ".join(kvs))
```